plugins/woocommerce/client/admin/client/marketplace/components/header-account/header-account.tsx

```diff
@@ -107,7 +129,7 @@ export default function HeaderAccount( {
 		<>
 			<DropdownMenu
 				className="woocommerce-layout__activity-panel-tab woocommerce-marketplace__user-menu"
-				icon={ avatar() }
+				icon={ dropdownTrigger() }
```

Comment:
To improve readability/cognitive load for `dropdownTrigger`, wdyt about moving the logic:

```suggestion
				icon={ 'wc-addons' !== page ? avatar() : dropdownTrigger() }
```

Reply:
I feel that having one function (`dropdownTrigger()`) that decides what to display is more readable. Otherwise, I might see `dropdownTrigger()` and wonder why it's not called in some contexts